packages/pds/src/api/app/bsky/graph/getAssertions.ts

```diff
@@ -96,6 +96,7 @@ export default function (server: Server) {
         limit,
         before,
         by: ref('assertion.createdAt'),
+        secondaryOrder: ref('assertion.uri'),
```

Comment:
fix for ordering issue for now. need a longer term fix that takes it into account w cursor: https://github.com/bluesky-social/atproto/issues/344